### DISCUSSION TITLE
security flaw?

### DISCUSSION THREAD

**Discussion by sunneydev** [sqids/sqids-javascript#24] [General]
do sqids protect from someone just collecting lots of different ids and just combining the characters from there to then match your "unique character set" ?

**Comment by 4kimov** [comment]
Not sure what _unique character set_ is referring to, but **security** and **sqids** do not go together in any combination, so the answer is probably no.

Edit: If you're referring to someone recreating the alphabet by simply observing IDs, yes it's possible. But the characters in the alphabet are not as important as the order of those characters.
